src/wasm/request.rs

```diff
@@ -254,6 +257,54 @@ impl RequestBuilder {
         self
     }
 
+    /// Set fetch credentials to 'same-origin'
+    ///
+    /// # WASM
+    ///
+    /// This option is only effective with WebAssembly target.
+    ///
+    /// The [request credentials][mdn] will be set to 'same-origin'.
+    ///
+    /// [mdn]: https://developer.mozilla.org/zh-CN/docs/Web/API/Request/credentials
```

Comment:
Mind changing these to `en-US` as the default link?

Reply:
Sorry , I didn't notice that.

Reply:
We need this feature in our project. Thanks for your patience.